File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-06/non-plagiarized/09/t06.kt
package IR

import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    val arrNum = IntArray(10)
    for (i in 0 until 10) {
        print("Read a number: ")
        arrNum[i] = sc.nextInt()
    }
    for (i in 9 downTo 0) {
        println(arrNum[i])
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-06/non-plagiarized/09/t06.ts
import { Scanner } from 'readline-sync';

const sc = new Scanner();

const arrNum: number[] = new Array(10);
for (let i = 0; i < 10; i++) {
  console.log('Read a number: ');
  arrNum[i] = sc.nextInt();
}

for (let i = 9; i > -1; i--) {
  console.log(arrNum[i]);
}

